charts/stable/otel-collector/values.yaml

```diff
@@ -0,0 +1,179 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: otel/opentelemetry-collector
+  # -- image tag
+  tag: 0.46.0
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- command is set to `otelcol` when using otel/opentelemetry-collector image but should be
+# changed to `otelcontribcol` if the image is changed to otel/opentelemetry-collector-contrib.
+# @default -- otelcol
+command:
+- "/otelcol"
+
+# -- args is set to the config file which is mounted via the configFileSecret or configFile
+# @default -- Set to the hard-coded volume mount.
+args:
+- "--config=/conf/otel-collector-config.yaml"
```

Comment:
This can be handled by the `hardcodedValues` solution as proposed in my comment on `common.yaml`